### PR TITLE
completed API validations user stories

### PR DESCRIPTION
**Pull request by Denise-Carlson**

### CODE REVIEW

File: app/controllers/dogs_controller.rb [hunk 1]
```diff
@@ -8,8 +8,12 @@ def index
   def create
     # Create a new dog
     dog = Dog.create(dog_params)
+    if dog.valid?
     render json: dog
+    else
+      render json: dog.errors, status:422
   end
+end
```
Comment (SunkissedQueen) [inline app/controllers/dogs_controller.rb:17]: The update and destroy controller methods (Lines 25 -29) should be above the private/strong params.

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Just a quick change on the arrangement of your controller methods.

File: app/controllers/dogs_controller.rb (hunk 1 above)
Comment (Denise-Carlson) [reply app/controllers/dogs_controller.rb:21]: I made the required changes. Can you approve ?

**Review by SunkissedQueen — APPROVED** [review]
Thanks for the correction. Remember to maintain indentation and spacing.

**Pull request merged**